### PR TITLE
Feature/pivotal tracker implementation

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: pivotal/api/features/Project.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      | enable_tasks                    | True       |
+      | enable_incoming_emails          | True       |
+      | public                          | False      |
+      | atom_enabled                    | False      |
+      | project_type                    | private    |
+      | enable_following                | True       |
+    And Validates schema with "post_project_schema.json"
+    And Sets a "GET" request to "/projects/ProjectObject.id"
+    And Sends request
+    And Should return status code 200
```
Comment (JuanCarlosMartinez4) [inline pivotal/api/features/Project.feature]: add a new line at end of code lines please

File: pivotal/jsonfiles/get_me_schema.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    "api_token",
+    "has_google_identity",
+    "accounts",
+    "projects",
+    "email",
+    "receives_in_app_notifications",
+    "created_at",
+    "updated_at"
+  ]
+}
```
Comment (JuanCarlosMartinez4) [inline pivotal/jsonfiles/get_me_schema.json]: add a new line at end of file please

File: pivotal/jsonfiles/post_project_schema.json [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    "atom_enabled",
+    "project_type",
+    "start_time",
+    "created_at",
+    "updated_at",
+    "account_id",
+    "current_iteration_number",
+    "enable_following"
+  ]
+}
```
Comment (JuanCarlosMartinez4) [inline pivotal/jsonfiles/post_project_schema.json]: add a new line at end of line please

File: trello/jsonfiles/put_boards_members.json [hunk 4]
```diff
@@ -185,4 +185,4 @@
     "members",
     "memberships"
   ]
-}
+}
```
Comment (JuanCarlosMartinez4) [inline trello/jsonfiles/put_boards_members.json]: add a new line at end of file please

File: .travis.yml [hunk 5]
```diff
@@ -13,7 +13,8 @@ install:
 script:
 - pytest test
 - behave trello/api/features --tags=~@wip --tags=~@defect
-- behave -f allure_behave.formatter:AllureFormatter -o ./reports ./trello/api/features/ --tags=~@defect
+- behave -f allure_behave.formatter:AllureFormatter -o ./reports ./trello/api/features/
+  --tags=~@defect
```
Comment (Amet015) [inline .travis.yml]: please go up this line

File: trello/api/features/Board.feature [hunk 6]
```diff
@@ -1,8 +1,9 @@
 Feature: Board
   As a regular user, it wants to manage a board, and creates a board.
 
-   @Smoke
+  @Smoke
   Scenario: Creates new board with a name
+    Given upload credential like "trello"
```
Comment (Amet015) [inline trello/api/features/Board.feature]: please change "like" by "as" in all features

**Review by Amet015 — APPROVED** [review]

File: pivotal/api/features/Me.feature [hunk 7]
```diff
@@ -0,0 +1,15 @@
+Feature: Credential
+  As a regular user, it wants to manage an account.
+
+  @Smoke
+  Scenario: Gets Credential
+    Given upload credential like "pivotal"
```
Comment (JuanCarlosMartinez4) [inline pivotal/api/features/Me.feature]: use **as** instead of **like**

File: pivotal/api/features/Project.feature [hunk 8]
```diff
@@ -0,0 +1,33 @@
+Feature: Project
+  As a regular user, it wants to manage an project.
+
+  @Smoke
+  Scenario: Adds a Project
+    Given upload credential like "pivotal"
```
Comment (JuanCarlosMartinez4) [inline pivotal/api/features/Project.feature]: use **as** instead of **like** and change in all scenarios

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Review by Amet015 — APPROVED** [review]

**Pull request merged**